x-pack/test/api_integration/apis/security/roles_bulk.ts

```diff
@@ -0,0 +1,427 @@
+                resources: ['*'],
+              },
+              {
+                application: 'kibana-.kibana',
+                privileges: ['space_all'],
+                resources: ['space:marketing', 'space:sales'],
+              },
+              {
+                application: 'logstash-default',
+                privileges: ['logstash-privilege'],
+                resources: ['*'],
+              },
+            ],
+            run_as: ['watcher_user'],
+            transient_metadata: {
+              enabled: true,
+            },
+          },
+          bulk_role_to_update_2: {
+            cluster: [],
+            indices: [],
+            applications: [
+              {
+                application: 'kibana-.kibana',
+                privileges: ['feature_dashboard.read', 'feature_dev_tools.all'],
+                resources: ['*'],
+              },
+              {
+                application: 'kibana-.kibana',
+                privileges: ['space_all'],
+                resources: ['space:observability', 'space:sales'],
+              },
+            ],
+            run_as: [],
+            metadata: {},
+            transient_metadata: {
+              enabled: true,
+            },
+          },
+        });
+      });
+    });
+
+    describe('Delete Roles', () => {
+      it('should delete the roles we created', async () => {
+        await supertest.delete('/api/security/role/bulk_role_1').set('kbn-xsrf', 'xxx').expect(204);
+        await supertest.delete('/api/security/role/bulk_role_2').set('kbn-xsrf', 'xxx').expect(204);
+        await supertest
+          .delete('/api/security/role/bulk_role_valid')
+          .set('kbn-xsrf', 'xxx')
+          .expect(204);
+        await supertest
+          .delete('/api/security/role/bulk_role_with_privilege_1')
+          .set('kbn-xsrf', 'xxx')
+          .expect(204);
+        await supertest
+          .delete('/api/security/role/bulk_role_with_privilege_2')
+          .set('kbn-xsrf', 'xxx')
+          .expect(204);
+        await supertest
+          .delete('/api/security/role/bulk_role_to_update_1')
+          .set('kbn-xsrf', 'xxx')
+          .expect(204);
+        await supertest
+          .delete('/api/security/role/bulk_role_to_update_2')
+          .set('kbn-xsrf', 'xxx')
+          .expect(204);
+
+        const emptyRoles = await es.security.getRole(
+          { name: 'bulk_role_1,bulk_role_2' },
+          { ignore: [404] }
+        );
+        expect(emptyRoles).to.eql({});
+        const rolesWithPrivileges = await es.security.getRole(
+          { name: 'bulk_role_with_privilege_1,bulk_role_with_privilege_2,bulk_role_valid' },
+          { ignore: [404] }
+        );
+        expect(rolesWithPrivileges).to.eql({});
+
+        const rolesToUpdate = await es.security.getRole(
+          { name: 'bulk_role_to_update_1,bulk_role_to_update_2' },
+          { ignore: [404] }
+        );
+        expect(rolesToUpdate).to.eql({});
+      });
```

Comment:
__question:__ sounds like something we should do in the `after` test hook instead since we're not really testing the delete API here?

Reply:
Addressed in https://github.com/elastic/kibana/pull/189173/commits/c91a43a2022d4f4f8452a4379d32de6db01bb4ea